dockers/docker-fpm-frr/frr/bgpd/templates/voq_chassis/peer-group.conf.j2

```diff
@@ -0,0 +1,23 @@
+!
+! template: bgpd/templates/voq_chassis/peer-group.conf.j2
+!
+  neighbor VOQ_CHASSIS_PEER peer-group
+  address-family ipv4
+{% if CONFIG_DB__DEVICE_METADATA['localhost']['type'] == 'ToRRouter' %}
+    neighbor VOQ_CHASSIS_PEER allowas-in 1
```

Comment:
This check will not be true for internal peers in VoQ chassis, can we remove this ?

Reply:
If one puts the VoQ chassis in the TOR role, then other templates would set up allowas-in on the eBGP connections. If it's not also set on the iBGP connections, any routes that do have a loop would be dropped by the iBGP peers. This was intended to avoid that routing asymmetry if ToRRouter is set. If it's not, there is no harm.